doc/stdlib/strings.rst

```diff
@@ -397,6 +411,10 @@
 
    Create a string from the address of a NUL-terminated UTF-32 string. A copy is made; the pointer can be safely freed. If ``length`` is specified, the string does not have to be NUL-terminated.
 
+.. function:: is_valid_utf32(s) -> Bool
```

Comment:
This is deprecated now, right? I guess we can leave the deprecated functions' docs in, but I'm not sure if we have a doctest to remind us to clear out obsolete docs anywhere.